src/navigation/tabs/shop/ShopHome.tsx

```diff
@@ -339,20 +339,25 @@ const ShopHome: React.FC<
             screenListeners={{
               tabPress: tab => {
                 if (tab.target) {
-                  setActiveTab(
-                    tab.target.includes(ShopTabs.GIFT_CARDS)
-                      ? ShopTabs.GIFT_CARDS
-                      : tab.target.includes(ShopTabs.BILLS)
-                      ? ShopTabs.BILLS
-                      : ShopTabs.SHOP_ONLINE,
-                  );
                   if (tab.target.includes(ShopTabs.BILLS)) {
+                    tab.preventDefault();
                     dispatch(
                       Analytics.track('Bill Pay - Clicked Bill Pay', {
                         context: 'Shop Tab',
                       }),
                     );
+                    navigation.navigate('Tabs', {
```

Comment:
I think that we can remove `Pay Bills`, in order to keep it simple.

Reply:
Thanks for the review Gustavo! `Pay Bills` will be removed from the Shop tab in a subsequent app release after existing Bill Pay users have been given a chance to get used to the new location of the Bill Pay feature.